index.Rmd

```diff
@@ -152,7 +152,7 @@ names(b)[20] <- "dist"
 # Create income brackets
 b_df <- as_data_frame(b)
 
-#median income brackets
+#median income brackets 
```

Comment:
What I would do here is to add a "tag": add a new column in the data frame and give each category a name, e.g. "less_than_20k", after filter( ). Then, we will not have to deal with a bunch of new data frames that occupy our memory space!
I used this method/tip in my project when I realized there are a lot of categories, it helped a lot!